profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.helpers.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @file
+ * Code for the NextEuropa Metatags module.
+ */
+
+/**
+ * Custom helper function.
+ *
+ * @return array
+ *   Returns the error message if the config of metatags does not comply.
+ */
+function _nexteuropa_metatags_check_metatag_config() {
+  $messages = array();
+
+  if ($config = metatag_config_load('global')) {
+    $meta_to_check = array('X-UA-Compatible', 'viewport');
+
+    foreach ($meta_to_check as $meta) {
+      if (!isset($config->config[$meta]) || (isset($config->config[$meta]) && empty($config->config[$meta]['value']))) {
```

Comment:
If the user has remove the meta X-UA-Compatible from his tpl there is no issue, the meta is now provided by the module. 
If the user has that meta tag in his template, he sould now remove it.
If the user has no meta tag in this template and config then he gets the error. Therefore for me it's perfect like this